nservicebus/upgrades/5to6.md

```diff
@@ -99,6 +99,8 @@ Configuration should be easy to apply to your code and easy to manipulate by han
 
 NServiceBus will automatically make correlated saga properties unique without the need for a explicit `[Unique]` attribute to be used. You can safely remove the attribute from your saga data types.
 
+### Saga persisters & finders
+Saga persisters `ISagaPersister` and finders `IFindSagas` introduce a new method parameter `SagaPersistenceOptions`. These options give access to the saga metadata and pipeline context within saga persisters and finders. This enables custom saga persisters and custom saga finders to have access to everything which is added to the context during message pipeline exectuion.
```

Comment:
would a before after sample be useful here? or at least a link to the existing doco that shows both versions?